src/visuals-api.d.ts

```diff
@@ -1029,6 +1029,14 @@ declare module powerbi {
     export interface FilterTypeDescriptor {
         selfFilter?: boolean;
     }
+
+    export const enum PendingChangesType {
+      "Filters",
```

Comment:
I would prefer to receive some human meaningful value instead of type number. That could help a lot when debugging a visual. Also, for this type, a single count naming or continuous action is better in my opinion.
`Filter = "FILTER"` or `Filtering = "FILTERING"` (uppercase is up to you). 
While "Filters" might be considered as some set of multiple values.